.stable

```diff
@@ -0,0 +1 @@
+v72.1.0
```

Comment:
Instead of having this file, I can try to work on something like writing directly to `setuptools/version.py`... But thought this was going to be the easiest approach.

This file helps to support fallback for workflows that don't use `tox` and/or `sdist` (e.g. downstream or when testing unpublished versions  with `pip install "setuptools @ git+https://github.com/abravalheri/setuptools@workaround-setuptools-scm"`).

Reply:
Unfortunately I did not found a way of implementing this particular methodology without having a "fallback" file. If I remove it then workflows that do not use the `wheel/sdist/tox` would result in a weird version of setuptools being reported (like `0.0.0` or `0.dev0+unknow`).